change/@fluentui-react-badge-cb6be496-2ef2-40b2-835f-f1e6e99710de.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 9 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27253&commitId=c1d99d8cf72163355679acbd107c6c82bc26880a&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1894 screenshots unchanged, 0 screenshots with different dimensions and 9 screenshots with visible difference.


  <details><summary><b>unknown</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.customSize+name+badge.normal.chromium.png|7| Changed|
|Avatar Converged.size+icon+badge+square.normal.chromium.png|42| Changed|
|Avatar Converged.size+image+badge.normal.chromium.png|8| Changed|
|Avatar Converged.size+inactive+badge.normal.chromium.png|26| Changed|
|Persona Converged.size-presence.normal.chromium.png|14| Changed|
|PresenceBadge Converged - OOF status.default - RTL.chromium.png|13| Changed|
|PresenceBadge Converged - OOF status.default.chromium.png|13| Changed|
|PresenceBadge Converged - sizes.default - RTL.chromium.png|7| Changed|
|PresenceBadge Converged - sizes.default.chromium.png|7| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>

Reply:
Looks like these are false positives? The changes are just minor color changes in the pixels.

Reply:
These are all intended changes, resolving conversation.